child_compassion/models/project_compassion.py

```diff
@@ -170,6 +170,9 @@ class CompassionProject(models.Model):
         'icp.sociological.activity', 'icp_socio_ado_act',
         string='Sociological activities (12+)', readonly=True
     )
+    reservation_id = fields.Many2one(
```

Comment:
You can as well add the field in the view of the project. You can add it like this :
```
<field name="reservation_id" context="{'default_project_id': active_id}"/>
```
so that the project_id field is already set when creating a reservation from the project.